charts/connect/values.yaml

```
@@ -21,7 +21,8 @@ connect:
     resources:
       limits:
         memory: 128Mi
-        cpu: 0.2
+      requests:
+        cpu: 0.2m
```

Change it to `200m` to make it equivalent to `0.2`

```suggestion
      requests:
        cpu: 200m
```

---

```suggestion
      requests:
        cpu: 0.2
```

I'll just use 0.2 for consistency